### PR TITLE
JAMES-2551 Dockerfile for Cassandra RabbitMQ JAMES

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/domainlist.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  software distributed under the License is distributed on an  
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       
+  KIND, either express or implied.  See the License for the    
+  specific language governing permissions and limitations      
+  under the License.                                           
+ -->
+
+<domainlist>
+    <domainnames>
+        <domainname>james.linagora.com</domainname>
```
Comment (aduprat) [inline dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/domainlist.xml]: s/linagora.com/apache.org

File: dockerfiles/compilation/java-8/compile.sh [hunk 2]
```diff
@@ -14,6 +14,7 @@ printUsage() {
 
 ORIGIN=/origin
 CASSANDRA_DESTINATION=/cassandra/destination
+CASSANDRA_RABBIT_DESTINATION=/cassandra-rabbit/destination
```
Comment (aduprat) [inline dockerfiles/compilation/java-8/compile.sh]: Why not `rabbitmq` in order to be consistent with product name?

File: README.adoc [hunk 3]
```diff
@@ -261,6 +261,60 @@ Add a link for the tika container in the above command line:
 
     $ docker run --hostname HOSTNAME -p "25:25" -p 80:80 -p "110:110" -p "143:143" -p "465:465" -p "587:587" -p "993:993" --link cassandra:cassandra --link elasticsearch:elasticsearch --link tika:tika --name james_run -t james_run
 
+=== Run James with Java 8 + Guice + Cassandra + ElasticSearch + RabbitMQ
+
+
+==== Requirements
+Built artifacts should be in ./dockerfiles/run/guice/cassandra/destination folder for cassandra.
```
Comment (aduprat) [inline README.adoc]: Missing `-rabbit` in path (and may be `-rabbitmq` depending on previous comment on product name).

**Comment by trantienduchn** [comment]
currently, there is a file at `src/site/markdown/server/install/guice-cassandra.md` do we need another for rabbit mq?

**Comment by chibenwa** [comment]
IMO we can wait to fully push RabbitMQ uses before advertising it on our website...

A README is more developer oriented and requires at least reading the sources: it is more suitable with today development status for Rabbit components.

**Comment by aduprat** [comment]
```
+ docker run -a stdout -a stderr --tmpfs=/tmp:rw,exec,size=1g --tmpfs=/james-project:rw,size=4g -v /var/run/docker.sock:/var/run/docker.sock --volumes-from=keys -v maven-cache-volume:/root/.m2 --volumes-from=git-publish-james-2551-docker-41c72551-e6d7-4c57-b82f-625f67652e25 --volume=/jpa/destination --volume=/cassandra/destination --volume=/spring/destination --name=james-parent-test-james-2551-docker-41c72551-e6d7-4c57-b82f-625f67652e25 --env JAMES_PARTIAL_BUILD_BASE_BRANCH=origin/master --env MVN_ADDITIONAL_ARG_LINE=-Ppartial-build james-parent-compile-james-2551-docker-41c72551-e6d7-4c57-b82f-625f67652e25 MERGE-james-2551-docker-41c72551-e6d7-4c57-b82f-625f67652e25
docker: Error response from daemon: No such container: keys.
```
test this please

**Comment by chibenwa** [comment]
This seems done

**Pull request closed without merging**